docs/getting-started/deploy-to-production.md

```diff
@@ -20,7 +20,7 @@ new project.
 After creating the project, head to the "Git" tab in the project settings,
 select the repository, the production branch (`main`), and the entrypoint file
 (`main.ts`). This will link the project to the repository and automatically
-deploy it to the internet. The project is now be available at
+deploy it to the internet. The project is now available at
```

Comment:
```suggestion
deploy it to the internet. The project will now be available at
```

This matches the tense of the previous sentence ("This will link...")

Reply:
you're absolutely right 👌🏼